train.py

```diff
@@ -164,4 +166,13 @@
 train_fea.to_csv('submission/nn_pred_{}_train.csv'.format(hparam.model_name),sep='\t',index=False,header=False)
 test_fea.to_csv('submission/nn_pred_{}_test.csv'.format(hparam.model_name),sep='\t',index=False,header=False)
 dev_fea.to_csv('submission/nn_pred_{}_dev.csv'.format(hparam.model_name),sep='\t',index=False,header=False)
-####################################################################################
\ No newline at end of file
+####################################################################################
+
+predict_label[['id','preds']] = test[['id','nn_preds']]
+
+# Calculate score
+score = Evaluation.calculate_score(predict_label,"data/testdata/test_df_label.csv","data/testdata/test_df.csv")
+print(("#Your score is %.4f")%(score*100.0))
+with open('ScoresLog.txt','a+') as scoreRecord:
```

Comment:
Suggestion: `'logs/CinScoresLog.txt'`, also update `README.md` to add `mkdir logs`.

Reply:
Change this to `Results/CINScore.txt` to save all users' score.